tests/test_env.py

```diff
@@ -46,7 +46,11 @@ def test_can_get_venv_paths_with_conflicting_default_scheme(
     assert get_scheme_names.call_count == 1
 
 
-@pytest.mark.skipif('posix_local' not in sysconfig.get_scheme_names(), reason='workaround for Debian/Ubuntu Python')
+SCHEME_NAMES = sysconfig.get_scheme_names()
+
+
+@pytest.mark.skipif('posix_local' not in SCHEME_NAMES, reason='workaround for Debian/Ubuntu Python')
+@pytest.mark.skipif('venv' in SCHEME_NAMES, reason='different call if venv is in scheme names')
```

Comment:
Isn't `venv` always in `SCHEME_NAMES`, since 3.11?

Reply:
How does this test pass, then? https://github.com/pypa/build/blob/e9adbf04f98745185ffb6552a7088a8ec12cfe2f/src/build/env.py#L346-L353 seems to trigger a path that would fail this test.

Reply:
(`get_paths.assert_called_once_with(scheme='posix_prefix', vars=mocker.ANY`, specifically)

Reply:
Sorry, I forgot to reply here. This test [isn't covered](https://app.codecov.io/gh/pypa/build/blob/henryiii%2Fbuild%3Ahenryiii%2Fci%2Fub/tests%2Ftest_env.py), so I assume it's always skipped.